tests/sync/util.js

```diff
@@ -159,6 +160,7 @@ const syncFromDisk = require('../../sync/syncFromDisk');
  * @property {string[]} syncIgnore
  * @property {string} rewriteUrl
  * @property {boolean=} enableNetworking
+ * @property {Object=} environment
```

Comment:
Nit, this is more precise:
```suggestion
 * @property {Record<string, string | null>=} environment
```